go.mod

```diff
@@ -57,3 +57,5 @@ require (
 replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+replace github.com/99designs/keyring => github.com/robert-zaremba/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
```

Comment:
In the docs we mentioned to all app developer to add the line `replace google.golang.org/grpc => google.golang.org/grpc v1.33.2`.

If we merge this PR, should all developers also add this line? If yes, I would maybe prefer to wait for 99designs to merge your PR and wait for the next release.

Reply:
The priority is to have a working software. 

If we don't want to have a replace line, then we can use my patch (or clone it into cosmos-sdk) and change the dependency -  which will require changing it in all files where 99designs/keyring package is used.